### PR TITLE
Add street functionality

### PR DESCRIPTION
**Pull request by slenrow**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,6 +17,8 @@
   "dependencies": {
     "axios": "^0.15.3",
     "body-parser": "^1.15.2",
+    "connect-mongo": "^1.3.2",
```
Comment (slenrow) [inline package.json:20]: @idandagan1 Ignore these additions for now. I was attempting to fix the session issues.

File: client/components/map/map.js [hunk 2]
```diff
@@ -1,41 +1,62 @@
...
-            navigator.geolocation.getCurrentPosition((position) => {
-                map.setCenter(new google.maps.LatLng(position.coords.latitude, position.coords.longitude));
-                map.setZoom(17);
-            }, () => {
-                map.setCenter(mapOptions.center);
-            });
-        } else { // Browser doesn't support Geolocation
-            map.setCenter(mapOptions.center);
-        }
+    shouldComponentUpdate(nextProps, nextState) {
```
Comment (idandagan1) [inline client/components/map/map.js:22]: like

File: client/views/my-streets/components/my-streets.js [hunk 3]
```diff
@@ -36,24 +47,36 @@ export default class MyStreets extends React.Component {
...
 
         return (
             <div className='n-mystreet'>
                 <div className='n-mystreet__add-street'>
-                    <GoogleSearch onSubmit={this.onSubmit} />
+                    <GoogleSearch onSubmit={this.handleSearchStreet} />
                     <button
                         type='submit'
                         className='n-mystreet__add-street-btn btn btn-sm n-btn-post'
+                        disabled={!placeId}
```
Comment (idandagan1) [inline client/views/my-streets/components/my-streets.js:71]: What's this for?

File: package.json (hunk 1 above)
Comment (idandagan1) [reply package.json:20]: Gotcha

**Review by idandagan1 — APPROVED** [review]
Great job man, you are a genius.

File: client/views/my-streets/components/my-streets.js (hunk 3 above)
Comment (slenrow) [reply client/views/my-streets/components/my-streets.js:71]: Disabling the add button until a user searches for a street in the search box.

**Pull request merged**